docs/reference/content/getting-started/async-quick-tour.md

```diff
@@ -0,0 +1,455 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Async Quick Tour"
+[menu.main]
+  parent = "Getting Started"
+  weight = 30
+  pre = "<i class='fa'></i>"
++++
+
+# MongoDB Async Driver Quick Tour
+
+The following code snippets come from the `QuickTour.java` example code
+that can be found with the [async driver
+source]({{< srcref "driver-async/src/examples/tour/QuickTour.java">}}).
+
+{{% note %}}
+See the [installation guide]({{< relref "getting-started/installation-guide.md#mongodb-async-driver" >}})
+for instructions on how to install the MongoDB Async Driver.
+{{% /note %}}
+
+## Going Async with Callbacks
+
+The MongoDB Async driver provides an asynchronous API that can leverage either Netty or Java 7's `AsynchronousSocketChannel` for fast and non-blocking IO.
+
+The API mirrors the new Synchronous API from the MongoDB Driver, but any methods that cause network IO take a `SingleResponseCallback<T>` and return immediately, where `T` is the type of response for the document.
+
+The `SingleResponseCallback<T>` interface requires the implementation of a single method `onResult(T result, Throwable t)` which is called when the operation has completed.  The `result` parameter contains the result of the operation, if successful. If the operation failed for any reason then the `t` contains the Throwable reason for the failure.
+
+{{% note class="important" %}}
+It's important to always check for errors in any `SingleResponseCallback<T>` implementation
+and handle them appropriately! Below the error checks are left out only for the sake of brevity.
+{{% /note %}}
+
```

Comment:
Would be nice to mention that SnigleResponseCallback is a SAM-type (Single Abstract Method) and so can be implemented by a lambda for users on Java 8.  And let's show one example using a lambda.